spec/lib/github_adapter_spec.rb

```diff
@@ -38,4 +38,38 @@
       subject.create_team(name, desc)
     end
   end
+
+  describe "#create_repo" do
+    it "calls POST with proper parameters" do
+      github_url = "https://api.github.com/orgs/chatto-hub-test2/repos"
+      expect(RestClient::Resource).to receive(:new).with(github_url, "example_user", "example_password").
+        and_return(double(post: {}.to_json))
+      subject.create_org_repo
+    end
+  end
+
+  describe "#fork_repo" do
+    let(:git) { double(add_remote: true, push: true, remote: 'git_remote') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.